### DISCUSSION TITLE
Can't edit doc from template

### DISCUSSION THREAD

**Discussion by mmospanenko** [outline/outline#6271] [Self Hosting]
New created document from template is not editable. 

**To Reproduce**
1. Create template
2. Create document from it
3. Try to edit body - it locked always (not about Edit mode)
4. ... but Title is editable

Interesting moment that after refreshing page this template disappears and I see empty body, then it can be editable again. Seems something in template blocks editing. Template in Ukrainian language.

**Expected behavior**
All templates should be editable.

**Screenshots**
<img width="479" alt="image" src="https://github.com/outline/outline/assets/2766211/58ba5d72-c0de-4b1e-8de2-18a410306c7d">

**Outline (please complete the following information):**
- Install: official docker, self hosted
- Version: v0.74.0

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome
 - Version: current

**Comment by surinara** [comment]
I solved it. My problem was nginx setup problem. I found the solution in the answer in the link below.
https://github.com/outline/outline/discussions/3677
> proxy_set_header Connection "Upgrade";
